.devcontainer/development/Dockerfile

```
@@ -25,9 +25,34 @@ ARG DOCS_DEST="/var/lib/OscarDocument/oscar/document"
+  fi
+
+  echo "apt install failed (attempt ${attempt}/3), retrying..."
+  apt-get clean || true
+  rm -rf /var/lib/apt/lists/*
+  if [ "$attempt" -eq 3 ]; then
+    exit 1
+  fi
+  sleep 5
+done
+EOF
+RUN chmod +x /usr/local/bin/apt-install
```

Hadolint (2.14.0)

[error] unexpected 'R'
expecting a new line followed by the next instruction

(DL1000)